pilot/pkg/xds/debug.go

```diff
@@ -823,3 +824,11 @@ func (s *DiscoveryServer) instancesz(w http.ResponseWriter, req *http.Request) {
 
 	_, _ = w.Write(by)
 }
+
+func (s *DiscoveryServer) networkgatewayz(w http.ResponseWriter, req *http.Request) {
+	w.Header().Add("Content-Type", "application/json")
+
+	network := s.globalPushContext().NetworkGateways()
```

Comment:
Should we just add this to the full push context handler?

Reply:
Huh, seems like I didn't ever push that PR.. let me check

Reply:
oh I just added sidecarz for SidecarScope. Well, anyhow, maybe we should have one for push context? We cannot just add the entire PushContext object since its both massive and recursive, but we can selectively add new fields so we don't need a new endpoint for every field? WDYT?

Reply:
SGTM.

Reply:
done